test/RulesTest.java

```diff
@@ -0,0 +1,45 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class RulesTest {
+
+    private RockRule rockRule;
+
+    @Before
+    public void setUp() throws Exception {
+        rockRule = new RockRule();
+    }
+
+    @Test
+    public void rockNotOneOfPlayersMove() {
+        assertEquals(null, rockRule.decideWinner(SYMBOL.PAPER, SYMBOL.PAPER));
+    }
+
+    @Test
+    public void rockBeatsScissors1() {
```

Comment:
I would say that `rockBeatsScissor1` & `rockBeatsScissor2` are testing the same logical assertion so in that case I would group them together in one test.